include/common/internal/app_ble_gap.h

```diff
@@ -1,30 +1,30 @@
 /**
```

Comment:
I contemplated about this, but the public function names deviate after the app_ble_gap_ prefix. For example app_ble_gap_scan_data_fetch_clear.

I think if I change the filename to app_ble_gap_state the function names should be updated and it will make them longer than necessary...